### DISCUSSION TITLE
consumer disappear without any exection

### DISCUSSION THREAD

**Discussion by koala775** [rabbitmq/rabbitmq-java-client#835] [General]
RabbitMQ version：3.6.5
Erlang version：19.2
Option：
Consumer disappear without any exection , accumulation

**Comment by michaelklishin** [comment]
Your consumers can disappear because their application (instances) stop or lose connectivity, or they run into [channel exceptions](https://www.rabbitmq.com/channels.html#error-handling). I'd start with inspecting [logs for lifecycle events](https://www.rabbitmq.com/logging.html#logged-events).

RabbitMQ 3.6.5 is from August 2016, and 3.6.x has gone entirely out of support [in May 2018](https://www.rabbitmq.com/versions.html). Time to move one beyond One True RabbitMQ Version™.

**Reply by koala775** [reply]
Thank you for replayimg.
